### PR TITLE
Alex post project management

### PR DESCRIPTION
**Pull request by AlexChadwickP**

### CODE REVIEW

File: content/blog/project-management-for-individuals/index.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+### 3. What will your development cycle look like?
+
+Whether it's well defined or not defined at all you inevitable have some sort of routine that you follow when programming. Write this down and also maybe consider if there are any improvements you could easily add to your routine. If you have trouble remembering when to commit for example (like me!) then try to set some sort time when you commit, like for example, when you've written 5 functions or when you've finished a step in your development cycle.
+
+### 4. Miscellaneous bits
+
+There are many other things that you can add to your methodology, but I chose to skip in mine. For example, a priority system, the fields you want your task types to have, do you want an effort system?, do you want to write a word document for a major feature? do you want to include notes in the task?
```
Comment (evelynhathaway) [inline content/blog/project-management-for-individuals/index.md:124]:
```suggestion
There are many other things that you can add to your methodology, but I chose to skip in mine. For example, a priority system, the fields you want your task types to have, do you want an effort system? Do you want to write a word document for a major feature? Do you want to include notes in the task?
```
Punctuation change.

An unordered list may also fit this sequence of questions.

File: content/blog/project-management-for-individuals/index.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+3. Writing tests: I prefer test driven development, so firstly I write unit and integration tests where I can. And I make sure they all fail!
+
+4. Implementation: Now that I've got the tests, I will program my implementation. For every test that succeeds I will create a commit.
+
+5. Refactoring / Documenting: Now that it's implemented and working, I do some refactoring and documenting where needed to make it easier to understand.
+
+6. CI: Now I will push all of my commits, and wait for the Continuous Integration pipeline to verify that all the testing, linting and building is successful
+
+7. (Optional) CD: This step is optional because I don't always deploy the changes I make immediately. If I'm fixing an important bug then this step will take place immediately after CI. If not then I'll wait to accumulate a few features to deploy my updated application
```
Comment (evelynhathaway) [inline content/blog/project-management-for-individuals/index.md:100]: I think it'd be great if you'd say Continuous Delivery somewhere in here, like in the CI section.

File: content/blog/project-management-for-individuals/index.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+### 3. What will your development cycle look like?
+
+Whether it's well defined or not defined at all you inevitable have some sort of routine that you follow when programming. Write this down and also maybe consider if there are any improvements you could easily add to your routine. If you have trouble remembering when to commit for example (like me!) then try to set some sort time when you commit, like for example, when you've written 5 functions or when you've finished a step in your development cycle.
+
+### 4. Miscellaneous bits
+
+There are many other things that you can add to your methodology, but I chose to skip in mine. For example, a priority system, the fields you want your task types to have, do you want an effort system?, do you want to write a word document for a major feature? do you want to include notes in the task?
+
+You can also work out how your pipelines are going to work. Do you want CI to be separate from your CD? Do you want the CI/CD pipeline run automatically or manually?
```
Comment (evelynhathaway) [inline content/blog/project-management-for-individuals/index.md:126]:
```suggestion
You can also work out how your pipelines are going to work. Do you want Continuous Integration (CI) to be separate from your Continuous Delivery (CD)? Do you want the CI/CD pipeline run automatically or manually?
```

I think the difference between CI and CD is subtle enough for many developers that saying the whole word once here would help some readers continue learning about the difference.

Props for adding the distinction between them here too! I love it.

**Review by evelynhathaway — COMMENTED** [review]
I love this post! I totally relate to your methodology and love the flexibility in options and ideas that your post provides for someone looking to up their project management game!

I left a few comments mostly on minor content items. Let me know what you think. 😊

**Comment by AlexChadwickP** [comment]
Thanks Evelyn! I've committed your suggestions, and appreciate the constructive feedback! I'll look out for similar improvements from now onwards, cheers!

**Review by evelynhathaway — APPROVED** [review]
Looks wonderful! Thanks so much!

**Pull request merged**